lib/alchemy/auth_accessors.rb

```diff
@@ -73,6 +72,16 @@ module Alchemy
   #     # config/initializers/alchemy.rb
   #     Alchemy.user_class_name = 'Admin'
   #
+
+  # Prefix with :: when getting to avoid constant name conflicts
+  def self.user_class_name
+    "::#{@@user_class_name}"
```

Comment:
I think it makes sense to `.freeze` the string here explicitly

Reply:
The hound said no freeze needed: https://github.com/AlchemyCMS/alchemy_cms/pull/1724#discussion_r382997756

Reply:
Yeah. Forgot that we already freeze all strings. Please ignore my comment